polyphemus/polyphemus.completion

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
```

Comment:
In general, are these `.completion` files auto-generated, or do we manually update? If we add a new command or change arguments, for example?